features/multiples/cautions_and_convictions.feature

```diff
@@ -0,0 +1,72 @@
+Feature: A person with cautions and convictions
+
+  @date_travel
+  Scenario: Discharge Order and Community Order
+    Given The current date is 28-04-2021
```

Comment:
Had to add this purely only because when running the whole test suite there seems to be a Date leaking into this test which makes one of the results to be in the future.

> This conviction was spent on 10 December 2020

When running the whole test suite:

> This conviction will be spent on 10 December 2020

I will add a note to investigate where the leak is coming from

Reply:
Maybe in another feature we didn't properly tag the scenario with `@date_travel` and then after it is finished it didn't travel back to current time. Good catch 👍